tasks/log_groups.yml

```diff
@@ -1,14 +1,15 @@
 ---
-
 - name: Ensure log groups {{ item.log_group_name }} exist 
   cloudwatchlogs_log_group:
     log_group_name: "{{ item.log_group_name }}"
-    retention: "{{item.log_retention|int}}"
-    tags: "{{item.log_group_tags}}"
-    aws_access_key: "{{ deploy_aws_access_key | default(aws_access_key | default(lookup('env','AWS_ACCESS_KEY_ID'))) }}"
-    aws_secret_key: "{{ deploy_aws_secret_key | default(aws_secret_key | default(lookup('env','AWS_SECRET_ACCESS_KEY'))) }}"
-    region: "{{ aws_region }}"
+    retention: "{{ item.log_retention|int }}"
+    tags: "{{ item.log_group_tags }}"
+    aws_access_key: "{{ deploy_aws_access_key | default(lookup('env','AWS_ACCESS_KEY_ID')) }}"
+    aws_secret_key: "{{ deploy_aws_secret_key | default(lookup('env','AWS_ACCESS_KEY_ID')) }}"
+    security_token: "{{ security_token | default(omit) }}"
```

Comment:
Whatever we end up doing `security_token` should follow a similar pattern as well, but also again this is a wider problem than just this role, everywhere that uses `deploy_aws_access_key` will need the same change...

Reply:
So just to be clear, the global `deploy_aws_acces_key` and `deploy_aws_secret_key` has to go. It causes this to fail when using an assumed role. This only effects London-architecture-based inventories for now so I'm only removing it from those, but a global user or auth token is a security risk. Easier to just use your own users and assume roles that have permissions to perform the action you need to perform.

Reply:
For sure it has to go, but you will have this issue in may places so better to do it as an explicit task rather than bit at a time so it is a just how it works, avoids doing one thing for one inventory and another, and/or breaking other use cases.